platform/conf/conf.go

```diff
@@ -184,12 +192,7 @@ func (u *UserData) Render() (*Conf, error) {
 			plog.Warningf("rendering Container Linux config: %s", report)
 		}
 
-		// ct still returns 2.0 configs. Convert to 2.1.
-		buf, err := json.Marshal(ignc)
-		if err != nil {
-			return nil, fmt.Errorf("serializing Container Linux config: %v", err)
-		}
-		return Ignition(string(buf)).Render()
+		c.ignitionV2 = &ignc
```

Comment:
The comment could have been clearer.  We were converting to 2.1 data structures, but the contents were left in 2.0 format.  With this change we're leaving everything as 2.0.